swim.go

```diff
@@ -182,7 +214,14 @@ func (s *SWIM) handle(msg pb.Message) {
 
 	switch msg.Payload.(type) {
 	case *pb.Message_Ping:
-		// handle ping
+		s.messageEndpoint.Send(msg.Address, pb.Message{
+			Seq: "",
+			Address: "",
+			Payload: &pb.Message_Ack{
```

Comment:
You must put your address(need to add member(me) into swim struct) in the address value.
Seq will be uuid. I think we should change the name of seq to id?

Reply:
+1